.github/workflows/test_bazel.yml

```diff
@@ -18,16 +18,21 @@ jobs:
       matrix:
         runner: [ ubuntu, windows, macos ]
         bazelversion: [ '7.1.2' ]
-        bzlmod: [true, false ]
+        bzlmod: [ true, false ]
+        toolchain_resolution: [ "" ]
         include:
           - runner: ubuntu
             bazelversion: '6.4.0'
-            bzlmod: true
+            # Not running Bazel 6 with bzlmod, because it doesn't support use_repo_rule in rules_jvm_external
+            bzlmod: false
           - runner: ubuntu
-            bazelversion: '6.4.0'
             bzlmod: false
+            toolchain_resolution: --incompatible_enable_proto_toolchain_resolution=true
+          - runner: ubuntu
+            bzlmod: true
+            toolchain_resolution: --incompatible_enable_proto_toolchain_resolution=true
     runs-on: ${{ matrix.runner }}-latest
-    name: Examples ${{ matrix.runner }} ${{ matrix.bazelversion }}${{ matrix.bzlmod && ' (bzlmod)' || '' }}
+    name: ${{ matrix.continuous-only && inputs.continuous-prefix || '' }} Examples ${{ matrix.runner }} ${{ matrix.bazelversion }}${{ matrix.bzlmod && ' (bzlmod)' || '' }} ${{ matrix.toolchain_resolution && ' (toolchain resolution)' || '' }}
```

Comment:
Remove all this continuous-prefix/continuous-only stuff?  It doesn't work on 28.x and I'm not sure if it could break anything

Reply:
(Not related to the current PR): Is this meant to work in v29.x? If no, how do we prevent that from getting pulled in when we cut the next branch?

Reply:
Done.

Reply:
Yes this works fine in main, and will work in 29